PodMergeExample/Podfile.lock

```diff
@@ -13,7 +13,7 @@ DEPENDENCIES:
   - UI (from `MergedPods/UI`)
 
 SPEC REPOS:
-  https://cdn.cocoapods.org/:
+  trunk:
```

Comment:
Do we need this change? I think we can continue to use the new Cocoapods CDN

Reply:
Uhmm weird, I've used the latest Cocoapods version so it should be using the CDN now. I'll check that.

Reply:
We're good with `trunk`. Take a look at this closed issue https://github.com/CocoaPods/CocoaPods/issues/9190.